SalesforceCommerceCloud/pwa-kit
# [V3] fix static Einstein recommendations

Small updates to some of the hooks that we use when calling Einstein to ensure that the Einstein API are getting the expected arguments.

This PR makes the following changes:
* For enc_user_id, removes an extra `$` character that prevented local storage lookup
* For usid, removes local storage lookup. This is because USID is stored in a cookie.

For both hooks, a whenReady function has been added to allow Einstein to wait on these values

Test steps:

After consuming these changes:
* Start the retail react app and navigate to several product pages
* As you navigate, observe that the various Einstein recommendations are updated
